### PR TITLE
Adgangsplatformen II

### PR DESCRIPTION
**Pull request by cableman**
#### Link to ticket

https://jira.itkdev.dk/browse/ER-1033

#### Description

Update the existing adgangsplatformen modules and code to support GUID and new Publizon API.

#### Screenshot of the result

No screenshot yet.

#### Additional comments or questions

Linked to https://github.com/eReolen/ereolengo/pull/61 and https://github.com/eReolen/ereolen/pull/39

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#### Link to ticket
```
Comment (rimi-itk) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: This is is a part of https://github.com/eReolen/ereolen/pull/39 and should not be included in this PR.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

**Comment by rimi-itk** [comment]
@cableman, you should check everything on the checklist or explain why you haven't checked everything.

**Comment by cableman** [comment]
Removed the checklist from template, as it do not make sens in this context

File: sites/all/modules/publizon/lib/client/PublizonUserClient.class.inc [hunk 2]
```diff
@@ -11,23 +11,64 @@
...
    *
-   * Validates that a given user is allowed to login to the publizon
-   * web-service.
+   * @param $card_number
+   *   The users card number/GUID/cpr.
+   * @param $retailer_id
+   *   The retailer id that belongs to the user.
+   *
+   * @return bool|string
+   *   FALSE if not found else the support id.
```
[the remark this answers is not in the document]
Comment (cableman) [reply sites/all/modules/publizon/lib/client/PublizonUserClient.class.inc:22]: Fixed

**Review by rimi-itk — APPROVED** [review]

**Comment by cableman** [comment]
Rebase on core and re-compile css
